service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -47,6 +47,10 @@ public boolean isExpertUIEnabled() {
         return ldClient.boolVariation("expert-ui", createLDUser(), false);
     }
 
+    public boolean isCaseUserAssignmentEnabled() {
```

Comment:
please add test for this one as it is done for rest of toggles

Reply:
done